### PR TITLE
Shaunda Cunningham

### PR DESCRIPTION
**Pull request by smcunning**

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+scan(/pattern/) { |match|  }
```
Comment (mikedao) [inline lib/school.rb:1]: When pushing to master, make sure you take out code that is extraneous.

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    @student_names = []
+  end
+
+  def add_student_name(student_name)
+    @student_names << student_name
+  end
+
+  def end_time
+    integer = @start_time.to_i + @hours_in_school_day
+    integer.to_s.concat(":00")
```
Comment (mikedao) [inline lib/school.rb:18]: Good

File: lib/school.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  def is_full_time?
+    if @hours_in_school_day > 4
+      true
```
Comment (mikedao) [inline lib/school.rb:23]: need to cover the reverse case that a short school day returns a false explicitly. We should be using an if/else here.

File: lib/school.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    end
+  end
+
+  def standard_student_names
+    capitalized_names = []
+    @student_names.each do |name|
+      capitalized_names << name.capitalize
+    end
+    p capitalized_names
```
Comment (mikedao) [inline lib/school.rb:32]: we're looking for this to return the new collection, not print it out and return it.

File: lib/school.rb (hunk 1 above)
Comment (smcunning) [reply lib/school.rb:1]: Got it! I wasn't sure if this was something that was used to grade the submitted code tbh! Will check for it next time.

File: lib/school.rb (hunk 3 above)
Comment (smcunning) [reply lib/school.rb:23]: Oof! Got it! Thanks!

**Pull request closed without merging**